EC_finetune/__main__.py

```diff
@@ -280,16 +295,31 @@ def main():
     if args.adapter_freeze_override:
         args.freeze_adapters = True
 
+    if args.sender_freeze_override:
```

Comment:
Nit: If I'm reading the correctly, here and below could be:

```
if args.sender_freeze_override:
  args.freeze_sender = args.sender_freeze_override:
```

Also another small readability nit, consider having the same naming conventions for the argument and override (`my_value` & `my_value_override` instead of `my_value` & `value_my_override`).